x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/latest_function_form.tsx

```diff
@@ -7,14 +7,20 @@
 import React, { FC } from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
-import { EuiComboBox, EuiFormRow } from '@elastic/eui';
+import { EuiButtonIcon, EuiCallOut, EuiComboBox, EuiCopy, EuiFormRow } from '@elastic/eui';
 import { LatestFunctionService } from './hooks/use_latest_function_config';
 
 interface LatestFunctionFormProps {
+  copyToClipboard: any;
```

Comment:
Can you please provide a type here?

Reply:
Good catch, rushed it, fixed in 422355e.